src/main/pages/workspace-admin/stacks.adoc

```diff
@@ -8,55 +8,55 @@ folder: workspace-admin
 ---
 
 
-[id="what-are-stacks"]
-== What Are Stacks
 
-A stack is a template of link:workspace-data-model.html[workspace configuration] and some meta-information like scope, tags, components, description, name and ID. Stacks are used by User Dashboard to make it easy to create workspaces and well as filter sample projects compatible with a chosen stack.
+[id="stack-overview"]
+== Stack overview
 
-Stacks are displayed in User Dashboard on Create a workspace page. You can filter them by type (single vs multi machine), scope (the ones displayed by default vs others), as well as search for a stack by keyword.
+A stack is a link:workspace-data-model.html[workspace configuration] template. Stacks are used to create workspaces in the `User Dashboard`. The stack includes meta-information such as scope, tags, components, description, name, and identification.  You can filter stacks by machine type and scope. The type is either single machine or multi machine.  You can also search for a stack by keyword. Stacks are displayed in the `User Dashboard` on the "Create a workspace" page.
 
-[id="how-stacks-are-used"]
-== How Stacks Are Used?
+See the link:creating-starting-workspaces.html[Creating and starting workspaces user guide] for more information.
 
-As stacks are workspace templates, they are used to create workspace when in User Dashboard. See: link:creating-starting-workspaces.html[User Guide: Creating and Starting Workspaces].
+[id="importing-community-supported-stacks-and-applications"]
+== Importing community supported stacks and applications
 
-[id="community-supported-stacks"]
-== Community Supported Stacks
+Che includes some stacks and sample applications that are pre-configured and tested. Stacks that are contributed by the Che community are not tested. Community stacks are located in the https://github.com/che-samples/community-stacks[community stacks GitHub repository].
 
-The list of stacks Che offers out of the box references tested stacks and sample applications with commands on how to build, run and debug them. There are also stacks contributed by the community which work fine with Che but do not go through daily tests. It is possible however that sample applications may be incompatible with certain stacks.
+Each directory has `${technology}-stack.json` and `${technology}-samples.json`.
 
-Community supported stacks and sample apps (if any) are located in https://github.com/che-samples/community-stacks[Community stacks GitHub repo].
+To import a stack, follow these steps:
 
-Each directory has `${technology}-stack.json` and `${technology}-samples.json`. If you want to import a chosen stack, follow the below instructions:
+.  Copy the content of the JSON.
```

Comment:
JSON is a format like, e.g. AsciiDoc. It needs qualification: "content of the JSON file", for example.